src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/CancellationProcessValidatorTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.access.entities.AccessControlResponse;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.idam.entities.UserInfo;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.RoleAssignment;
+import uk.gov.hmcts.reform.wataskmanagementapi.config.LaunchDarklyFeatureFlagProvider;
+import uk.gov.hmcts.reform.wataskmanagementapi.config.features.FeatureFlag;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.utils.CancellationProcessValidator;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.singletonList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.lenient;
+
+@ExtendWith(MockitoExtension.class)
+class CancellationProcessValidatorTest {
+
+    private CancellationProcessValidator cancellationProcessValidator;
+    AccessControlResponse mockAccessControlResponse;
+    @Mock
+    private RoleAssignment mockedRoleAssignment;
+
+    @Mock
+    LaunchDarklyFeatureFlagProvider launchDarklyFeatureFlagProvider;
+
+    @BeforeEach
+    void setUp() {
+        cancellationProcessValidator = new CancellationProcessValidator(launchDarklyFeatureFlagProvider);
+        mockAccessControlResponse = new AccessControlResponse(
+            new UserInfo("id", "idamId", List.of("Admin"), "surname", "email", "familyName"),
+            singletonList(mockedRoleAssignment)
+        );
+    }
+
+    @ParameterizedTest
```

Comment:
we have a naming convention for our tests. please review. It should follow "Should_ExpectedBehavior_When_StateUnderTest"